lib/xcprofiler/derived_data.rb

```diff
@@ -26,7 +26,7 @@ def by_pattern(pattern)
           raise DerivedDataNotFound, 'Any matching derived data are not found'
         end
 
-        derived_data.max_by { |data| data.updated_at }
+        derived_data.select { |data| !data.zero? }.max_by { |data| data.updated_at }
```

Comment:
I'd prefer to use `derived_data.reject { |data| data.zero? }`.

Reply:
That’s nice :+1: